### DISCUSSION TITLE
Git add error - error: 'main/' does not have a commit checked out ?

### DISCUSSION THREAD

**Discussion by 1natalie** [leightonavantgardehaus/.github#6] [Q&A] [answered]
I am trying to push a rails 6 app to heroku. I am following these steps:

git init
git add .
git commit -m “My first commit”
heroku create
git push heroku master
when I run command: git add . I get this error:
git add .
error: ‘main/’ does not have a commit checked out
fatal: adding files failed

**Comment by TL8N** [comment, marked as answer]
There may be directory called Main, If you can find directory called Main , there may be .git folder hidden into it.

If there is nothing else in Main directory, you may try deleting it and then re-run commands.

Lastly, try to go to view on windows and check hidden files and folders, .git folder should show in that dir, delete and try running the command


Hopefully it should work!!
